client/src/main/java/io/cloudsoft/winrm4j/client/WinRmClient.java

```diff
@@ -226,6 +231,10 @@ private synchronized WinRm createService() {
 
         BindingProvider bp = (BindingProvider)winrm;
         bp.getRequestContext().put(BindingProvider.ENDPOINT_ADDRESS_PROPERTY, endpoint.toExternalForm());
+        if (requestTimeoutInSeconds != null) {
+            //BindingProviderProperties.REQUEST_TIMEOUT
+            bp.getRequestContext().put("com.sun.xml.internal.ws.request.timeout", requestTimeoutInSeconds * 1000);
```

Comment:
Oops no `Repeater` out of brooklyn, a simple sleep (100-200ms) should work for now.